test/auth.test.js

```diff
@@ -0,0 +1,51 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import models from '../models';
+import 'babel-polyfill';
+import server from '../app';
+
+chai.use(chaiHttp);
+
+describe('AUTHENTICATION TEST SUITE', () => {
+  before(() => models.sequelize.sync({ force: true }));
+
+  describe('JWT AUTHENTICATION', () => {
+    it('should return a token on successful registration',
+      (done) => {
+        const newUser2 = {
+          username: 'ebenezer',
+          email: 'ebenezer@gmail.com',
+          password: 'secretstuff',
+        };
+        chai.request(server)
```

Comment:
Could you refactor these tests with `async-await`?

You could also add extra expect statements;
- Checking that the token type is a string 
- Checking that the length of the token is greater than 1.

Those extra expects are just to add extra color to the test spec instead of having just not `undefined` as the only assertion.